design-system/materials/internals/definition.yaml

```diff
@@ -1109,7 +1115,7 @@ decisionGroupsByTheme:
         padding-for-stamp:
           choice: '4px 12px'
         padding-for-stamp-as-condensed:
-          choice: '0 var(--spacing-20)'
+          choice: '2px var(--spacing-10)'
```

Comment:
@FilPob Do you think we will be using this `2px` value elsewhere? If so, maybe it make sense to create a new design token like `spacing-05` or something like that instead.